modules/multi-architecture-enabling-64k-pages.adoc

```diff
@@ -0,0 +1,84 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/multi-architecture-configuration.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="multi-architecture-enabling-64k-pages_{context}"]
+
+= Enabling 64k pages on the {op-system-first} kernel 
+
+You can enable 64k memory page in the {op-system-first} kernel on the 64-bit ARM compute machines in your cluster. The 64k page size kernel specification can be used for large GPU or high memory workloads. This is done using the Machine Config Operator (MCO) which uses a machine config pool to update the kernel To enable 64k page sizes, you must create a machine config pool for AMD64 and a machine config pool for ARM64.
+
+[IMPORTANT]
+====
+Using 64k pages is exclusive to 64-bit ARM architecture compute nodes or clusters installed on 64-bit ARM machines. If you configure the 64k pages kernel on a machine config pool using 64-bit x86 machines, the machine config pool and operator will degrade.
+====
+
+.Prerequisites: 
+* You installed the OpenShift CLI `oc`
+* You created a cluster with compute nodes of different architecture on one of the supported platforms
+
+.Procedure: 
+
+. Add a label to the node you want to assign as the infra node with a specific label: 
+//not sure if this part was needed
+[source,terminal]
++
+----
+$ oc label node <node_name> <label>
+----
+
+. Create a machine config pool that contains the worker role that uses the ARM64 architecture and the `64k-pages` role:
+[source,yaml]
++
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+metadata:
+  name: worker-64k-pages
+spec:
+  machineConfigSelector:
+    matchExpressions:
+      - key: machineconfiguration.openshift.io/role
+        operator: In
+        values:
+        - worker
```

Comment:
should we advise to patch this config to remove the  `worker` role? i assume it wouldn't include the amd64 nodes as well because of the arch label selector?

Reply:
Hmm, I'm not too sure, @aleskandro Do you know this one?

Reply:
I'm not sure I got you 100% @Prashanth684 .

Users should not remove the `worker` element in the MachineConfig's selector's value: the child MCP `worker-64k-pages` should still make use of the MachineConfigurations bound to the built-in `worker` role and add the ones bound to the custom role we are documenting.

Instead, yes, the nodeSelector will take care of binding only arm64 nodes to the custom MCP

Reply:
ok..my concern was that it would not filter correctly if all nodes in the worker pool was also included, but looks like that's not the case so all good.